packages/react-native-attestation/package.json

```diff
@@ -15,18 +15,10 @@
     "react-native-attestation.podspec"
   ],
   "scripts": {
-    "test": "jest",
-    "typecheck": "tsc --noEmit",
-    "lint": "eslint .",
-    "build": "yarn run clean && bob build",
+    "build": "echo 'building react-native-attestation 🏗️' && yarn run clean && bob build && echo 'react-native-attestation built ✅'",
     "clean": "rimraf ./build && del-cli android/build ios/build",
-    "compile": "tsc",
-    "prepack": "bob build",
-    "release": "release-it",
-    "example": "yarn --cwd example",
-    "build:android": "cd example/android && ./gradlew assembleDebug --no-daemon --console=plain -PreactNativeArchitectures=arm64-v8a",
-    "build:ios": "cd example/ios && xcodebuild -workspace AttestationExample.xcworkspace -scheme AttestationExample -configuration Debug -sdk iphonesimulator CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++ GCC_OPTIMIZATION_LEVEL=0 GCC_PRECOMPILE_PREFIX_HEADER=YES ASSETCATALOG_COMPILER_OPTIMIZATION=time DEBUG_INFORMATION_FORMAT=dwarf COMPILER_INDEX_STORE_ENABLE=NO",
-    "bootstrap": "yarn example && yarn install && yarn example pods"
```

Comment:
We removed the example folder a while back so none of these are used. We don't use release-it